src/symbol.ts

```diff
@@ -0,0 +1,61 @@
+import {Sprite, Texture} from "pixi.js";
+import {gsap} from "gsap";
+import {getRandomInt} from "./utils";
+
+interface SlotSymbol {
+    startBounce(): Promise<void>;
+    moveOneSlot(): Promise<void>;
+    endBounce(): Promise<void>;
+    setSymbol(name: Texture): void;
+}
+
+export default class Symbol extends Sprite implements SlotSymbol {
+
+    private _startPosition: { x: number, y: number };
+
+    constructor(texture: Texture, x: number, y: number, size: number) {
+        super(texture);
+        this.x = x;
+        this.y = y;
+        this.anchor.set(0.5);
+        this.scale.x = this.scale.y = Math.min(size / this.width, size / this.height);
+        this._startPosition = {x, y};
+    }
+
+    public startBounce(): Promise<void> {
+        return new Promise((resolve) => {
+            gsap.to(this, {
+                duration: getRandomInt(1, 2),
```

Comment:
I understand, it's just "playing around", but in next steps let's have the symbol configurable: all the timings and distances (and any other configs, if any) should be provided into Symbol's constructor